UPGRADING.md

```diff
@@ -15,6 +15,45 @@ end
 
 See [#957](https://github.com/intridea/grape/pull/957) for more information.
 
+#### Replace error_response with error! in rescue_from blocks
```

Comment:
Just to be clear, this isn't required, right? I mean the old code still works, so maybe you want to call that out?